kratos/processes/output_process.h

```diff
@@ -0,0 +1,121 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher (https://github.com/philbucher)
+//
+
+
+#if !defined(KRATOS_OUTPUT_PROCESS_H_INCLUDED )
+#define  KRATOS_OUTPUT_PROCESS_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "process.h"
+
+
+namespace Kratos
+{
+///@addtogroup ApplicationNameApplication
+///@{
+
+///@name Kratos Classes
+///@{
+
+/// Short class definition.
+/** Detail class definition.
+*/
+class OutputProcess : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of OutputProcess
+    KRATOS_CLASS_POINTER_DEFINITION(OutputProcess);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    OutputProcess() : Process() {}
+
+    /// Copy constructor.
+    OutputProcess(OutputProcess const& rOther) = delete;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    /// Assignment operator.
+    OutputProcess& operator=(OutputProcess const& rOther) = delete;
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    virtual bool IsOutputStep()
+    {
+        return true;
```

Comment:
Maybe we should throw an error in here.

Reply:
I would not
Some processes can always print, seems ok to me